protos/cassie/sockets.proto

```diff
@@ -1,20 +1,35 @@
 syntax = "proto3";
 import "runme/runner/v2/runner.proto";
+import "google/rpc/code.proto";
 
 option go_package = "github.com/jlewi/cloud-assistant/protos/gen/cassie";
 
+// Represents socket-level status (e.g., for auth, protocol, or other errors).
+message SocketStatus {
```

Comment:
@jlewi lemme now if these protos changes work for you.

Reply:
Does the connect proto have its own protos that are equivalent to the grpc ones? If it does should we use that? 
https://connectrpc.com/docs/protocol/#error-codes

If your encoding a message in addition to the code isn't there a status proto that has both of those? Should we just use that either in place of SocketStatus Or nested inside SocketStatus
https://connectrpc.com/docs/go/errors/

Reply:
I believe the `Connect` error codes are the same. I'll check. I'll see if it makes sense to use the `Connect` ones instead.

> WDYT of my proposal in #71 to use the tuple (cellID, RunID, sequenceID)?

Yep, I'm working on that next.